README.markdown

```diff
@@ -241,6 +241,11 @@ You can provide your own lookup stubs using the built in `::Timezone::Lookup::Te
 
     ::Timezone.lookup(-11, 11)
     => Timezone::Error::Test: missing stub
+    
+You can provide a default lookup stub, in that case error is not going to be raised if a stub is missing.
```

Comment:
Maybe adjust phrasing a little: "You can also provide a fallback lookup, which will be returned if you query an un-stubbed lookup value."